### PR TITLE
Updating database and save to handle the new metadata input

### PR DESCRIPTION
**Pull request by Celinet21**
## PR Description
- New requirements from customer mean the input now includes a sample_metadata file, containing information (this new information has been edited into Models.py)

- Save now will take in a sample_metadata file, which contains `Sample Name,Cohort Name,Batch Name,Flowcell.Lane,Library ID,Platform,Centre of Sequencing,Reference Genome,Type,Description`

  - A new cohort is made if the cohort name does not exist yet (note that we assume this file has 1 cohort, so only check the first data row for its name).

- Save also now takes a batch_description and cohort_description string. Add this if the save command is used on 1 new batch or 1 new cohort respectively. It will add this description anytime a new batch / cohort is detected and made.

I tested this on the new large input given, but please help me test yourself and see if it works.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] Ensure the CI test suite passes (coming soon...).
 - [x] Make sure your code lints (coming soon...).
 - [x] `CHANGELOG.md` is updated (if >=v1.0.0)
 - [x] `README.md` is updated

### CODE REVIEW

**Review by lwratten — COMMENTED** [review]
I'll pull this from your branch and have a go at testing 👍

**Comment by Kento-Bento** [comment]
I can test this but only after 6pm today sorry

File: falcon_multiqc/commands/save.py [hunk 1]
```diff
@@ -2,54 +2,91 @@
...
+                    cohort_row = Cohort(
+                        id=split[1],
+                        type=split[7],
+                        description=cohort_description
+                    )
+                    session.add(cohort_row)
+
+                # Get cohort id / name from the first data row (assuming the metadata is for 1 cohort).
+                if not cohort_id:
+                    cohort_id = split[1]
```
Comment (Kento-Bento) [inline falcon_multiqc/commands/save.py:49]: Just a suggestion, 
image
And still have the same functionality, not sure which is better, up to you~

**Review by Kento-Bento — APPROVED** [review]

File: falcon_multiqc/commands/save.py (hunk 1 above)
Comment (Celinet21) [reply falcon_multiqc/commands/save.py]: Thanks I like that, done! There's also a suggestion button in Github btw if you wanna make suggestions.

**Pull request merged**